cms/envs/common.py

```diff
@@ -915,3 +915,26 @@
         "type": "date"
     }
 }
+
+# The configuration for Studio user profiles
+PROFILE_CONFIGURATION = {
```

Comment:
@andy-armstrong I thought you were going to just import this variable from lms' common.py file.